cypress/support/step_definitions/anonymous_user_view_entity_registry_metadata_HTML.js

```diff
@@ -18,27 +18,35 @@
 //      | Location of SPARQL endpoint      |
 //      | Description of available formats |
 
+import {Then, When} from 'cypress-cucumber-preprocessor/steps';
 
-when(/an anonymous user dereferences the base URI for the registry specifying mediatype text\/html/, (tableData) =>{
-	cy.log('-- anonymous_user_view_entity_registry_metadata_HTML.js --')
-	
-	cy.get('@registryName').then((registryName) => {
-		cy.visit('/registry/' + registryName)
-	})
+When(/an anonymous user dereferences the base URI for the registry specifying mediatype text\/html/, (dataTable) => {
+  cy.log('-- anonymous_user_view_entity_registry_metadata_HTML.js --');
 
-})
+  let attributeArray = dataTable.rawTable;
+  cy.wrap(attributeArray).as('attributeNames');
 
-then('they see metadata related to the entity registry regarding:', (dataTable) =>{
-	const attributeArray = dataTable.rawTable;
+  cy.get('@registryName').then((registryName) => {
+    const createRegistryEndpoint = '/registry/' + registryName;
+    cy.request({
+      url: createRegistryEndpoint,
+      method: 'GET',
+      headers: {
+        'content-type': 'text/html',
```

Comment:
A const TEXT_HTML is defined in another file. Better to use a const here too? Or define some global constants?

Reply:
Also do we by content-type mean 'accept'?

Reply:
Fixed